app/controllers/items_controller.rb

```diff
@@ -0,0 +1,40 @@
+class ItemsController < ApplicationController
+  def index
+  end
+
+
+  def new
+    @item = Item.new
+  end
+
+  def show
+    @item = Item.find(params[:id])
+    @lists = @list.items
+  end
+
+  def create
+    @item = Item.new(item_params)
```

Comment:
This will work, however, is it being associated with the user that created it?

How might we do something like that using the built in methods the has_many association gives us?